internal/repository/copy.go

```diff
@@ -64,27 +64,23 @@ func (r *Copy) CopyFile(
 	}
 	defer func() { _ = in.Close() }()
 
-	out, err := r.appFs.Create(dst)
+	si, err := r.appFs.Stat(src)
 	if err != nil {
 		return err
 	}
-	defer func() {
-		if e := out.Close(); e != nil {
-			err = e
-		}
-	}()
```

Comment:
This one is weird.  It looks like it was trying to throw an error on exit if there was a problem closing the file, but it looks like it will only successfully do that if the function is already throwing an error.  If the function is returning successfully, this func does nothing.  And if the func is already returning an error, and there is also a problem, this will potentially mask the "other" problem?

Since it's a cleanup function, I'm going to assume we actually don't give a single, solitary shit if it cannot close the file, and drop the value on the floor instead.